runtime/onert/backend/cpu/ops/AttentionLayer.cc

```diff
@@ -0,0 +1,383 @@
+  //   Shape: [n_batch, n_tokens, d_model]
+  //   Data: float*
+  // Weight tensors: _wq, _wk, _wv, _wo
+  //   Shape: [d_model, d_model] (assuming d_q = d_k = d_v = d_model)
+  //   Data: float*
+
+  // Define the output shape for Q and K projections
+  nnfw::cker::Shape proj_output_shape({n_batch, n_tokens, d_model});
+  nnfw::cker::FullyConnectedParams fc_params{};
+
+  // Q Projection
+  std::vector<float> q_proj_buf(n_batch * n_tokens * d_model);
+  nnfw::cker::FullyConnected(fc_params, getShape(_input), getBuffer<float>(_input), getShape(_wq),
+                             getBuffer<float>(_wq), getShape(nullptr) /*bias_shape=*/,
+                             /*bias_data=*/nullptr, proj_output_shape, q_proj_buf.data());
+
+  // K Projection
+  std::vector<float> k_proj_buf(n_batch * n_tokens * d_model);
+  nnfw::cker::FullyConnected(fc_params, getShape(_input), getBuffer<float>(_input), getShape(_wk),
+                             getBuffer<float>(_wk), getShape(nullptr) /*bias_shape=*/,
+                             /*bias_data=*/nullptr, proj_output_shape, k_proj_buf.data());
+
+  // V Projection
+  std::vector<float> v_proj_buf(n_batch * n_tokens * d_model);
+  nnfw::cker::FullyConnected(fc_params, getShape(_input), getBuffer<float>(_input), getShape(_wv),
+                             getBuffer<float>(_wv), getShape(nullptr) /*bias_shape=*/,
+                             /*bias_data=*/nullptr, proj_output_shape, v_proj_buf.data());
+
+  // 2. RoPE
+
+  // _cos, _sin
+  //   Shape: [ n_batch, n_tokens, d_head ] (assume n_batch=1, n_tokens=1)
+  //   DataType: float*
+
+  // 2.1 nullcheck (Validation moved to configure())
+
+  // 2.2  inputs: (input, cos, sin)
+  // RoPE expects 4D tensor for input and sin/cos tables.
+  // The projection output (proj_out) is logically reinterpreted:
+  //     from [ n_batch, n_tokens, d_model ]
+  //     to   [ n_batch, n_head, n_tokens, d_head ]
+  // This reinterpretation is valid as d_model = n_head * d_head and n_tokens = 1.
+  nnfw::cker::Shape rope_in_shape({n_batch, n_head, n_tokens, d_head});
+  // Extend cos, sin to 4D
+  nnfw::cker::Shape sin_cos_shape({1, n_batch, n_tokens, d_head});
+
+  // 2.3 output buffer:
+  nnfw::cker::Shape rope_out_shape({n_batch, n_head, n_tokens, d_head});
+  std::vector<float> q_rope_buf(rope_out_shape.FlatSize());
+  std::vector<float> k_rope_buf(rope_out_shape.FlatSize());
+
+  nnfw::cker::RoPEMode rope_mode = nnfw::cker::RoPEMode::kGptNeox;
+
+  // 2.4 Call cker::RoPE
+  nnfw::cker::RoPE<float>(rope_mode, rope_in_shape, k_proj_buf.data(), sin_cos_shape,
+                          getBuffer<float>(_sin), sin_cos_shape, getBuffer<float>(_cos),
+                          rope_out_shape, k_rope_buf.data());
+  nnfw::cker::RoPE<float>(rope_mode, rope_in_shape, q_proj_buf.data(), sin_cos_shape,
+                          getBuffer<float>(_sin), sin_cos_shape, getBuffer<float>(_cos),
+                          rope_out_shape, q_rope_buf.data());
```

Comment:
Can't we fill `_cos` and `_sin` without input (or model weight) by using values from `n_heads`, `n_tokens`, etc from above variables? I don't know do we really need these tensors.